apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts

```diff
@@ -237,37 +252,35 @@ export const PdfEditorLabelWrapper = styled.div`
   .badge {
     margin: 0px 0px 0px 3px;
   }
-`;
+`; // no spacing design token for 4.5rem and 3px
```

Comment:
If the Figma design does't specify a token, then lets not worry about it and just do what you did, but leave the comment out.

Reply:
I did commented these before figma design I will remove them. Sorry somehow i missed to remove comments